stdlib/int32.mli

```diff
@@ -125,6 +125,12 @@ external to_int : int32 -> int = "%int32_to_int"
    during the conversion.  On 64-bit platforms, the conversion
    is exact. *)
 
+val to_int_unsigned : int32 -> int
```

Comment:
I would prefer the `Int32.unsigned_to_int` naming. While the result in `int` cannot be negative you are not treating the `int` as an unsigned `int`.

Reply:
(Ah sorry in fact you are) but I don't think you should. As a said these operations are mainly useful for indexing arrays, this why they should return options if the exceed the positive range of `int`s.

Reply:
I don't think I am, did I make a mistake? In any case renamed `to_int_unsigned -> unsigned_to_int` as suggested.